modules/stanford_events_importer/src/Plugin/Field/FieldWidget/StanfordEventsImporterAPIURLFieldWidget.php

```diff
@@ -38,6 +38,13 @@ class StanfordEventsImporterAPIURLFieldWidget extends LinkWidget {
    */
   protected $cache;
 
+  /**
+   * {@inheritdoc}
+   */
+  public static function defaultSettings() {
+    return [];
+  }
```

Comment:
That was it eh?

Reply:
yup. i could've added a schema.yml to define those settings, but since we don't use those settings in the widget, there was no point.

Reply:
Won't this strip out the placeholder settings though?